src/poh_recorder.rs

```diff
@@ -53,7 +53,7 @@ impl PohRecorder {
         // This guarantees PoH order and Entry production and banks LastId queue is the same.
         let mut poh = self.poh.lock().unwrap();
         let tick = poh.record(mixin);
-        self.bank.register_entry_id(&tick.id);
+        // don't register an entry
```

Comment:
Outside of this PR, this comment should seem weird and out of place. Not registering here should be expressed as a unit-test in this module, not a code comment. Considering the number of times we've broken this, it should be a unit-test with an especially long comment justifying its existence and reminding @rob-solana what'll happen if he removes it again. :)

Reply:
I resemble that remark